cli/medperf/commands/result/create.py

```diff
@@ -68,15 +68,10 @@ def prepare(self):
         # If not running the test, redownload the benchmark
         self.benchmark = Benchmark.get(self.benchmark_uid)
         self.ui.print(f"Benchmark Execution: {self.benchmark.name}")
-        self.dataset = Dataset.from_generated_uid(self.data_uid)
-        if not self.run_test:
-            self.out_path = results_path(
-                self.benchmark_uid, self.model_uid, self.dataset.uid
-            )
-        else:
-            self.out_path = results_path(
-                self.benchmark_uid, self.model_uid, self.dataset.generated_uid
-            )
+        self.dataset = Dataset.get(self.data_uid)
+        self.out_path = results_path(
+            self.benchmark_uid, self.model_uid, self.dataset.uid
```

Comment:
I will dig more into this, but for now it seems that we will get results tmp uids like this: "1_2_None" since we are using the dataset server uid